relay-kafka/src/limits.rs

```diff
@@ -0,0 +1,103 @@
+use std::cmp;
+use std::collections::HashMap;
+use std::fmt;
+use std::sync::Arc;
+use std::sync::atomic::{AtomicU64, Ordering};
+use std::time::{Duration, Instant};
+
+use parking_lot::RwLock;
```

Comment:
Any specific reason to reach for parking lot?

Reply:
mostly to avoid poisoning errors, don't think anything is in a bad state if we panic halfway through